### ISSUE TITLE
`Github` object sometimes not there?

### ISSUE THREAD

**Issue by arokem** [yeatmanlab/AFQ-Browser#146]
Report from @jyeatman says that he's now getting: 

```
Traceback (most recent call last):
  File "/home/jyeatman/anaconda2/bin/afqbrowser-publish", line 4, in <module>
    __import__('pkg_resources').run_script('AFQ-Browser==0.2.dev0', 'afqbrowser-publish')
  File "/home/jyeatman/anaconda2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 738, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/jyeatman/anaconda2/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1499, in run_script
    exec(code, namespace, namespace)
  File "/home/jyeatman/anaconda2/lib/python2.7/site-packages/AFQ_Browser-0.2.dev0-py2.7.egg/EGG-INFO/scripts/afqbrowser-publish", line 18, in <module>
    url = gh.upload(args.target, args.reponame)
  File "/home/jyeatman/anaconda2/lib/python2.7/site-packages/AFQ_Browser-0.2.dev0-py2.7.egg/afqbrowser/gh_pages.py", line 39, in upload
    g = gh.Github(uname, upass)
AttributeError: 'module' object has no attribute 'Github'
```

**Comment by arokem** [comment]
Hey @jyeatman : what happens if you open a Python or IPython terminal and type: 

    import github as gh
    gh.Github

Same thing?

**Comment by jyeatman** [comment]
Same thing

**Comment by jyeatman** [comment]
This works fine on my mac laptop

**Comment by arokem** [comment]
Yeah - I suspect this has to do with different versions of PyGithub.

**Comment by arokem** [comment]
Probably resolved by #152
